src/core/scripting.c

```diff
@@ -818,7 +818,7 @@ mSCRIPT_DEFINE_STRUCT(mScriptTextBuffer)
 	mSCRIPT_DEFINE_STRUCT_METHOD(mScriptTextBuffer, clear)
 	mSCRIPT_DEFINE_DOCSTRING("Set the number of rows and columns")
 	mSCRIPT_DEFINE_STRUCT_METHOD(mScriptTextBuffer, setSize)
-	mSCRIPT_DEFINE_DOCSTRING("Set the posiiton of the cursor")
+	mSCRIPT_DEFINE_DOCSTRING("Set the position of the cursor")
```

Comment:
Consider splitting this off and also the whitespace fixes into a separate PR, especially since those are the only changes for this file.